server/node/src/server.ts

```diff
@@ -12,10 +14,27 @@ import {
   createSetupTokenWithSampleDataForPayPal,
 } from "./paypalServerSdk";
 
+const envFilePath = join(__dirname, "../../../", ".env");
+config({ path: envFilePath });
+
+const CLIENT_STATIC_DIRECTORY =
```

Comment:
Can you explain why we need this `CLIENT_STATIC_DIRECTORY` env var? Presumably if the developer moves the client directory I think the app not working is a fair expectation and the developer should be responsible for fixing that. Just wondering if it would be simpler to not offer this env var 🤔